specs/devexperience/DevExperience.md

```diff
@@ -0,0 +1,181 @@
+1.	What kicks in the MSIX Package tooling?
+2.	Can developers opt-out of the pri generation?
+3.	How does building a Sparse Package look?
+    - A Sparse Package is a Win32 app that has identity, but isn’t using MSIX.
+    - This means there is a package.appxmanifest and a SxS manifest
+4.	What to do with the Windows store /start menu assets that aren’t used in unpackaged applications
+5.	How does an existing UWP developer move forward and use this?
+6.	How does an existing WPF or WinForms developer move forward and use this?
+
+## 2.0 – Definitions
+- Reunion – The effort to unify the Win32 and UWP platforms
+- Reunion Tooling – MSBuild and VisualStudio tooling used to build Reunion apps.
+- Workflow – The steps a developer will take in Visual Studio to be successful using Reunion.
+- WAP – A project used to generate an .msix or .msixbundle.
+- Sparse Package – An unpackaged Win32 app with package Identity via registration of an .msix.
+
+## 3.0 Guiding Principles
+Below is a list of some guiding principles to ground this proposal
+1.	Easy for developers to understand.
+2.	Easy for developers to change between the four different app model types:
+	- Unpackaged
+    - Sparse Package
+    - Packaged Desktop
+    - UWP
+3.	Easy for developers to migrate their existing Win32 and UWP apps.
+a.	Maintain compat with Microsoft.AppxPackage.targets
+
+## 4.0 Key Design Elements
+In order to satisfy principles 2 and 3 above, there are some key design elements which need to be taken into account:
+1.	A single MSBuild property, `WindowsPackageType`, will enable developers to easily change their application type. It is an enum value with the following four options: 
+    - `None`
+    - `Sparse`
+    - `Desktop`
+    - `Universal`
+    - Potential future options: `Framework` (for framework package)
+2.	The package.appxmanifest file cannot require changes between Sparse, Packaged Desktop, and UWP. The tooling will generate the appropriate appxmanifest.xml based on the project being built.
+3.	There must be an Item Template for the package.appxmanifest
+4.	The default splash screen and start menu assets will be provided by the tooling. The same behavior of preventing store submissions for apps which use these assets will still exist. 
+5.	A migration tool will be provided for migrating projects
+    - Work for this has started with the https://github.com/dotnet/try-convert tool.
+    - This tool needs to be updated for .vcxproj.
+    - This tool will update code, project files, and manifest files
+    - Update nuget references to WinUI3 supported packages (i.e. Win2D & WCT)
+6.	The existing Manifest designer tool in VS must work for these projects
+7.	The existence of a Package.appxmanifest file will enable Packaged Desktop apps by default.
+8.	The Reunion NuGet package will provide tooling for generating the Sparse Package and Dynamic Dependencies registration code. (link to existing issues)
```

Comment:
Should also allow the user to disable the build task and customize that code if wanted.